source/_posts/2024-01-13-leaky-bucket.markdown

```diff
@@ -0,0 +1,108 @@
+---
+title: "Leaky Bucket Rate Limiting"
+kind: article
+created_at: 2024-01-13 19:40:00 UTC
+author: Shivam Singhal
+layout: post
+---
+Hmm…leaky bucket? The first time I heard these words, it made me chuckle. I thought what kind of name is this for a rate limiting strategy.
+
+But upon exploring it, it’s rather a very simple, elegant, and popular strategy used in a lot of production environments.
```

Comment:
Prefer active voice, since the rest of the article is like that.
```suggestion
But upon exploring it, I found it to be a rather simple and elegant strategy used in a lot of production environments.
```